deployments/mainnet/weth/configuration.json

```diff
@@ -7,7 +7,7 @@
   "governor": "0x6d903f6003cca6255d85cca4d3b5e5146dc33925",
   "pauseGuardian": "0xbbf3f1421d886e9b2c5d716b5192ac998af2012c",
   "storeFrontPriceFactor": 0.5,
-  "targetReserves": "5000000e6",
+  "targetReserves": "5000000e18",
```

Comment:
Bumping the target reserves was required for scenarios; not sure what the actual value for this should be